### ISSUE TITLE
Use AllTheBlocks API to fetch peers rather than scraping the web-page

### ISSUE THREAD

**Issue by qoole** [guydavis/machinaris#840]
https://github.com/guydavis/machinaris/blob/b0037c41a56a901fbf452b93c350290ca83b5312/api/commands/websvcs.py#L343

The request_peers function can be re-written as:

```PYTHON
def request_peers(blockchain, debug=False):
    alltheblocks_blockchain = globals.get_alltheblocks_name(blockchain)
    url = "https://api.alltheblocks.net/{0}/peer/recent?amount=10".format(alltheblocks_blockchain)
    app.logger.info("Requesting node peers for {0} from {1}".format(blockchain, url))
    response = requests.get(url)
    data = response.json()
    peers = []
    for peer in data:
        peers.append(peer["host"] + ":" + str(peer["port"]))
    return peers
```

This is both more polite on the alltheblocks website, and less risk of it breaking due to a website change. It is also tunable for number of peers to request should you want to expose that option...

**Comment by guydavis** [comment]
Awesome!  I'm not sure how I missed that in the ATB API before.  As always, I really appreciate the second set of eyes on Machinaris code.

**Comment by guydavis** [comment]
Available now in the Machinaris [v0.8.7 release](https://github.com/guydavis/machinaris/releases/tag/0.8.7).
